src/stats.wsgi

```diff
@@ -47,14 +47,13 @@ def application(environ, start_response):
     output = output.replace("{total}", str(row[0]))
 
     # by status
-    query.execute("SELECT status, COUNT(*) FROM tasks GROUP BY status")
+    query.execute("SELECT COUNT(*) FROM tasks WHERE status = {0}".format(STATUS_SUCCESS))
     row = query.fetchone()
-    while row:
-        if int(row[0]) == STATUS_SUCCESS:
-            output = output.replace("{success}", str(row[1]))
-        elif int(row[0]) == STATUS_FAIL:
-            output = output.replace("{fail}", str(row[1]))
-        row = query.fetchone()
+    output = output.replace("{success}", str(row[0]))
+
+    query.execute("SELECT COUNT(*) FROM tasks WHERE status = {0}".format(STATUS_FAIL))
```

Comment:
I am not sure what you mean.  Until now all other items on this page are done the way I rewrote this -> execute some query, and with its response replace some tag in some string. These two I rewrote did it in loop, but when some value was not used, it was not replaced.